lib/EditorJs.tsx

```diff
@@ -62,13 +63,21 @@ class EditorJsContainer extends React.PureComponent<Props> {
     onChange(api, newData)
   }
 
+  handleReady = () => {
+    const { onReady } = this.props
+    if (!onReady) return
```

Comment:
This repo use code style with brackets for `if (...) { ... }` conditions

Reply:
Thank you, you are right. I've updated this line.